pkg/resource/notebook_instance/testdata/test_suite.yaml

```diff
@@ -0,0 +1,358 @@
+             - operation: StartNotebookInstance
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopped_after_update.yaml"
+       - name: "ReadOne=StoppedAfterUpdateHuman"
+         description: "Testing how readone handles the Stopped state when stopped by human for an update, StoppedByControllerMeta should 
+         not be there."
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopped_after_update_human.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopped_after_update_ack.json"
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopped_after_update.yaml"
+       - name: "ReadOne=StoppedBeforeUpdateACK"
+         description: "Testing how readone handles the Stopped state when it's stopped right before an update. StoppedByControllerMeta should be in 
+         UpdatePending."
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopped_before_update_ack.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopped_before_update_ack.json"
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopped_before_update_ack.yaml"
+       - name: "ReadOne=StoppingDeleting"
+         description: "Testing how readone handles the Stopping stopping state before going into deleting"
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopping_deleting.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopping_deleting.json"
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopping_deleting.yaml"
+       - name: "ReadOne=StoppedUpdate"
+         description: "Testing how readone handles the Stopping stopping state before going into Update"
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopping_update.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopping_update.json"
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopping_update.yaml"
+       - name: "ReadOne=UpdatingUser"
+         description: "Testing how readone handles the Updating status when a user stops the Notebook. StoppedByControllerMeta should
```

Comment:
where is there? StoppedByControllerMeta should not be in status?

Reply:
edited it